### PR TITLE
2015 Officers Update

### PR DESCRIPTION
**Pull request by ElijahCaine**
@tschuy @phillels @lucywyman @rettigs Check this out.

### CODE REVIEW

**Comment by lucywyman** [comment]
Added my stuff -- +1

File: content/people/elijahvoigt.mkd [hunk 1]
```diff
@@ -4,18 +4,17 @@ rank: 1
...
+    irc: pop
     image: /img/people/elijahvoigt.png
 
 ---
 
-Elijah is a second year Computer Science student at OSU and a software devloper
-at the [Open Source Lab][osl]. In his spare time he writes games in [lua][lua] 
-and plans LUG meetings. He also enjoys long walks on the beach and writing 
-bios in the third person. 
+Elijah is a second year Computer Science student at OSU and a software
```
Comment (tschuy) [inline content/people/elijahvoigt.mkd]: third year?

File: content/people/elijahvoigt.mkd [hunk 2]
```diff
@@ -4,18 +4,17 @@ rank: 1
...
+developer at the [Open Source Lab][osl]. In his spare time he writes webapps in
```
Comment (edunham) [inline content/people/elijahvoigt.mkd]: is "webapps" a word?

**Comment by lucywyman** [comment]
Added Elsie.

@rettigs @ElijahCaine @tschuy

**Comment by tschuy** [comment]
+1

**Comment by rettigs** [comment]
+1

**Comment by lucywyman** [comment]
+1

**Pull request merged**